docs/user-guide/cloud-storage.rst

```diff
@@ -43,8 +45,8 @@ Rasa supports using `S3 <https://aws.amazon.com/s3/>`_ ,
     variable to the path of that key file.
 
 * Azure Storage
-    Azure is supported using the legacy ``azure-storage-blob`` package (v 2.1.0),
-    which you can install with ``pip install -I azure-storage-blob==2.1.0``.
+    Azure is supported using the ``azure-storage-blob`` package,
+    which you can install with ``pip install azure-storage-blob``.
```

Comment:
Not sure which version is the latest one. As @erohmensing is offline, maybe @melindaloubser1 can help?

Reply:
Ooh I do not know this off the top of my head, but editing times show that the one in master was edited 2 months ago vs the 1.10.x one a year ago.

Reply:
Did you mean the actual version of azure-storage-blob?

Reply:
> Did you mean the actual version of azure-storage-blob?

Yeah the version was added on master, but some other changes were made on 1.10.x. 

I guess, I'll keep the master version then. Thanks.